### DISCUSSION TITLE
Label custom components in integration panel

### DISCUSSION THREAD

**Discussion by TheZoker** [home-assistant/frontend#11233] [Other feature requests]
## The request

The new integration panel is really nice:
![Screenshot_2020 02 19_18h18m46s_003_](https://user-images.githubusercontent.com/1368405/74857748-6d02c100-5344-11ea-9563-4000a8d050df.png)

One thing I'm currently missing is, which integration is a custom component and which is loaded from core. 
This would debugging things make easier in my opinion. (maybe someone forgot, that for testing purpose he placed a custom component - which overwrites a core integration -  in his HA and since the last update it's causing issues).

## The alternatives

Don't label custom components

**Comment by balloob** [comment]
Another thing we should add is that if it's a custom integration, we should link to the documentation of the custom integration. Now all links go to our domain.

**Comment by alandtse** [comment]
@balloob Did you want a separate issue to cover the wrong link for custom integrations? I was about to open the issue but found your comment.

**Comment by frenck** [comment]
@alandtse There is already an issue open for that: #4929
